app/globals.css

```
@@ -196,6 +206,124 @@ body.font-loading *::after {
+.composer-textarea {
+  width: 100%;
+  max-height: 160px;
+  min-height: 24px;
+  line-height: 1.4;
+  padding: 10px 6px;
+  background: transparent;
+  border: none;
+  outline: none;
+  resize: none;
+  overflow: auto;
+  font: inherit;
```

Biome (2.1.2)

[error] Unexpected shorthand property font after line-height



(lint/suspicious/noShorthandPropertyOverrides)